src/Main/LedBar.cpp

```diff
@@ -110,6 +110,17 @@ void LedBar::setColor(uint8_t red, uint8_t green, uint8_t blue) {
   }
 }
 
+/**
+ * @brief Sets the hue of all LEDs
```

Comment:
I haven't worked with the HSV color model yet, but I would expect that we want to set this on a per LED base (and not for all).

Reply:
@lschloetterer

Reply:
You're right, having the possibility to individually set the LEDs would be more dynamic.

However I see now that the code for the LED CO2 handler isn't in the arduino sketch anymore, but moved into the library in the AgStateMachine class. So having a custom handler isn't possible at the moment without modifying the library code anyway.

I'm not sure if there is a good solution to provide the possibility to have a custom LED handler function, I'd have to look into this when I have some more time

Reply:
@lschloetterer I am closing this PR for now. Please feel free to open a new PR at a later stage which is compatible with the current firmware.